opentdf-example-remote-ers.yaml

```diff
@@ -0,0 +1,111 @@
+mode: all
+modeoptions:
+  remoteERSUrl: http://localhost:8181
```

Comment:
What are your thoughts if we moved this configuration underneath `SDKConfig`. Trying to think how we could layer in other options for our internal products as well. 

https://github.com/opentdf/platform/blob/c36624cbefd8643cad6f01603046fa060ae06724/service/internal/config/config.go#L45-L58